src/main/java/com/iemr/hwc/data/location/DistrictBranchMapping.java

```diff
@@ -85,20 +86,32 @@ public class DistrictBranchMapping {
 	@Column(name = "LastModDate", insertable = false, updatable = false)
 	private Timestamp lastModDate;
 
+	@Column(name = "latitude")
+	@Expose
+	private Double latitude;
+	@Column(name = "longitude")
+	@Expose
+	private Double longitude;
+	@Column(name = "active")
+	@Expose
+	private Boolean active;
+
 	@Transient
 	private OutputMapper outputMapper = new OutputMapper();
 
 	public DistrictBranchMapping() {
 	}
 
 	
-	public DistrictBranchMapping(Integer districtBranchID,String villageName, Integer govtLGDSubDistrictID, Integer govtLGDVillageID
+	public DistrictBranchMapping(Integer districtBranchID,String villageName, Integer govtLGDSubDistrictID, Integer govtLGDVillageID, Double latitude, Double longitude, Boolean active
```

Comment:
You should not modify an existing constructor, it might lead to exceptions. Please, create a new one.

Reply:
Done @Shrey475